mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FC, useEffect } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { AppScreenName, PaginationDefaultValue } from '~/common/enums/enums';
+import { GroupsCreateRequestDto } from '~/common/types/types';
+import { Button, Input, Text } from '~/components/common/common';
+import {
+  PermissionsTable,
+  UsersTable,
+} from '~/components/uam-groups-config/components/components';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  usePagination,
+  useSelectedItems,
+} from '~/hooks/hooks';
+import { groupsCreationActions, uamGroupEditActions } from '~/store/actions';
+import { groupCreateClient } from '~/validation-schemas/validation-schemas';
+
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from './common/constants/create-group-default.constants';
+import { styles } from './styles';
+
+const UAMConfigureGroup: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
+  const { group } = useAppSelector((state) => state.uamGroupEdit);
+
+  const { users, permissions } = useAppSelector(
+    (state) => state.uamGroupCreation,
+  );
+
+  const { page: usersPage, handlePageChange: handleUserPageChange } =
+    usePagination();
+
+  const {
+    page: permissionsPage,
+    handlePageChange: handlePermissionsPageChange,
+  } = usePagination();
+
+  const { control, handleSubmit, errors, reset } =
+    useAppForm<GroupsCreateRequestDto>({
+      defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+      validationSchema: groupCreateClient,
+    });
+
+  const groupName = control._formValues.name;
```

Comment:
I mean you shouldn't access value through private property

Reply:
Can you give me a hint please, I don't crearly understand how to do this now?

Reply:
You can look how it's done on frontend side, if it's not our case then you can try useWatch

Reply:
on frontend they use 'control._formValues.name'

Reply:
@revenets try to use payload from `handleSubmit` callback and create only one custom submit callback for `handleSubmit` (`handleCreateOrEdit` for example)